commit_check.sh

```diff
@@ -41,6 +41,10 @@ done
 rm shafile.txt  >/dev/null 2>&1
 
 echo "No commits exist with valid formatting."
+echo "At least one commit in the pull request must use the angular format."
+echo "This means that at least one commit must match this regex exactly (Note especially the whitespace):"
+echo "(feat|fix|docs|style|refactor|perf|test|chore)#([0-9])*: .*"
```

Comment:
is this even correct?

it accepted 

feat(#1141): blah blah

which isn't what this regex describes.

also, people don't work in regexes, we should put it more human readable.

```
feat(#0000): your commit message here

where feat is the commit type. the options are feat|fix|docs|style|refactor|perf|test|chore
and 0000 is the github issue number

note there is no whitespace between feat and the issue number, and there is a colon after the issue number
```

Reply:
`feat(#1141): blah blah` is a valid commit message.